docs/en/observability/infrastructure-threshold-alert.asciidoc

```diff
@@ -8,8 +8,8 @@ resource or a group of resources within your infrastructure.
 Additionally, each rule can be defined using multiple
 conditions that combine metrics and thresholds to create precise notifications and reduce false positives.
 
-. To access this page, go to *{observability} > Metrics*.
-. On the *Inventory* page or the *Metrics Explorer* page, click *Alerts > Infrastructure*.
+. To access this page, go to **{observability}** -> **Metrics**.
```

Comment:
It looks like the name of menu item is changed. There is no **Metrics** item in menu, instead it is **Infrastructure**. 
<img width="443" alt="Screenshot 2022-08-16 at 13 32 02" src="https://user-images.githubusercontent.com/69037875/184869962-d587c619-55b5-4e9b-bb5c-93ed44c943ab.png">

Reply:
Thanks for letting us know! I'll address this separately. See https://github.com/elastic/observability-docs/issues/2097.